CHANGELOG.md

```
@@ -0,0 +1,100 @@
+
+### Added
+- MediaQueryLogger for improved logging
+- Queries::fromDir method
+
+## Earlier versions
+
+Please refer to the git history for changes in earlier versions.
+
+[Unreleased]: https://github.com/ray-di/Ray.MediaQuery/compare/0.16.0...HEAD
+[0.16.0]: https://github.com/ray-di/Ray.MediaQuery/compare/0.15.1...0.16.0
+[0.15.1]: https://github.com/ray-di/Ray.MediaQuery/compare/0.15.0...0.15.1
```

markdownlint-cli2 (0.17.2)

Link and image reference definitions should be needed
Unused link or image reference definition: "0.15.1"

(MD053, link-image-reference-definitions)